packages/react-examples/src/react/Dialog/docs/DialogOverview.md

```diff
@@ -1 +1,5 @@
 A dialog box (`Dialog`) is a temporary pop-up that takes focus from the page or app and requires people to interact with it. It’s primarily used for confirming actions, such as deleting a file, or asking people to make a choice.
+
+### React 17 event delegation
+
+Changes to event delegation in React 17 can cause issues with event bubbling when using Dialog. [See `Layer` documentation](#/controls/web/layer) for details on how to resolve eventing issues with this control in React 17.
```

Comment:
I think it'd be nice to include the link to the React blog post for all components rather than just Layer, though I think it makes sense to keep the specific `eventBubblingEnabled` fix just in the Layer documentation.